### PR TITLE
LanguageStep

### LINKED ISSUE #68: (SP:2) Implement language selection
User Story: #12 , #13 

## Description

Implement language selection at `Language Step` at stepper.

## Design
Image
Image

### PR DESCRIPTION
**Pull request by HalynaProlubshchykova**

### CODE REVIEW

**Review by grbvtsk — APPROVED** [review]

**Review by SeredenkoDS — APPROVED** [review]
I'm not yet too experienced in React to leave a meaningful review sadly, but overall looks alright! Approving.

**Review by olehpona — APPROVED** [review]

**Review by MrAK-dev — APPROVED** [review]

**Review by SachkovaYuliia — APPROVED** [review]

File: src/containers/tutor-home-page/language-step/LanguageStep.jsx [hunk 1]
```diff
@@ -1,19 +1,116 @@
...
-      <Box sx={styles.rigthBox}>
-        <Box>Language here</Box>
+    <Container sx={styles.container}>
+      <ImgContainer>
+        <Img src={img} sx={styles.img} />
+      </ImgContainer>
+      <RightBox sx={styles.rigthBox}>
+        <Title variant='h6'>
+          Please select the language in which you would like to study and
+          cooperate.
```
Comment (vlad-khrychov) [inline src/containers/tutor-home-page/language-step/LanguageStep.jsx]: Use translation for all strings of front end

File: src/containers/tutor-home-page/language-step/LanguageStep.jsx [hunk 2]
```diff
@@ -1,19 +1,116 @@
...
+        <MobileImgContainer>
+          <Img src={img} />
+        </MobileImgContainer>
+        <AutocompleteStyled
+          inputValue={inputValue}
+          onChange={(event, newValue) => setValue(newValue)}
+          onInputChange={(event, newInputValue) => setInputValue(newInputValue)}
+          options={languages}
+          renderInput={(params) => (
+            <TextField {...params} label='Your native language' />
```
Comment (vlad-khrychov) [inline src/containers/tutor-home-page/language-step/LanguageStep.jsx]: Use translation for all strings of front end

File: src/containers/tutor-home-page/language-step/LanguageStep.jsx [hunk 3]
```diff
@@ -1,19 +1,116 @@
...
+  marginBottom: '16px'
+})
+
+const MobileImgContainer = styled(Box)(({ theme }) => ({
+  display: 'none',
+  [theme.breakpoints.down('sm')]: {
+    display: 'block',
+    textAlign: 'center'
+  }
+}))
```
Comment (vlad-khrychov) [inline src/containers/tutor-home-page/language-step/LanguageStep.jsx]: I like the styled components approach, but move this code to another file to make the `LanguageStep.jsx` shorter.

File: src/containers/tutor-home-page/language-step/LanguageStep.jsx [hunk 4]
```diff
@@ -1,19 +1,116 @@
...
+  width: '100%',
+  maxWidth: '400px',
+  [theme.breakpoints.down('md')]: {
+    maxWidth: '300px'
+  },
+  [theme.breakpoints.down('sm')]: {
+    maxWidth: '100%',
+    marginBottom: '16px'
+  }
+}))
```
Comment (vlad-khrychov) [inline src/containers/tutor-home-page/language-step/LanguageStep.jsx]: Do you think maybe this styled `Img` should be shared with other steps?

**Review by vlad-khrychov — CHANGES_REQUESTED** [review]
Discuss with the team, maybe your styled components can be used for other steps as well.

**Review by vlad-khrychov — CHANGES_REQUESTED** [review]
let's rename the folder into `shared` and delete `.styles` from file names.

**Review by vlad-khrychov — APPROVED** [review]

**Pull request merged**